samples/helloworldasync/helloworld.csproj

```diff
@@ -0,0 +1,7 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+	<LangVersion>7.1</LangVersion>
```

Comment:
Ok, I see now what the default ends up being and that this is needed. Can you fix the indentation (LangVersion should not be nested lower than OutputType). Then I can merge this. Thanks!

Reply:
Done.